### PR TITLE
website: latest news section

### PR DESCRIPTION
**Pull request by captainill**
Adds latest news section which includes upcoming vault webinar

### CODE REVIEW

**Comment by captainill** [comment]
Screencap screencapture-localhost-4567-1489806528516

**Comment by captainill** [comment]
Screencap vault-latestnews

**Comment by briankassouf** [comment]
@captainill There is one merge conflict here

**Comment by captainill** [comment]
@briankassouf ah Thanks! Didn't realize. I resolved this.

File: website/source/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -22,6 +22,8 @@
 @import '_header';
 @import '_footer';
 @import '_buttons';
+@import '_sidebar';
```
Comment (briankassouf) [inline website/source/assets/stylesheets/application.scss]: @captainill Was this PR supposed to contain a sidebar file?

Comment (jefferai) [inline website/source/assets/stylesheets/application.scss]: @briankassouf There is a `website/source/layouts/_sidebar.erb` file, which I would guess is that one.

**Pull request merged**